File: MoreDakka/Startup.Db.fs
namespace MoreDakka

open Microsoft.AspNet.Identity
open Microsoft.AspNet.Identity.EntityFramework
open Microsoft.AspNet.Identity.Owin
open System.Data.Entity
open MoreDakka.Models

type MyDbInitializer() =
    inherit DropCreateDatabaseAlways<IdentityContext>()

    let createUserManager context =
        new UserManager<ApplicationUser>(new UserStore<ApplicationUser>(context))

    let createRoleManager context =
        new RoleManager<IdentityRole>(new RoleStore<IdentityRole>(context))

    let init context =
        use userManager = createUserManager context
        use roleManager = createRoleManager context

        let adminRoleName = "Admin"
        let adminUser = ApplicationUser(UserName = "admin", Email = "jamez226@hotmail.com")

        if not <| roleManager.RoleExists(adminRoleName) then
            roleManager.Create(IdentityRole(adminRoleName)) |> ignore

        userManager.Create(adminUser, password = "123456") |> ignore
        userManager.AddToRole(adminUser.Id, adminRoleName) |> ignore

    override this.Seed(context) =
        init context
        base.Seed(context)

File: MoreDakka/App_Start/Startup.Db.cs
using System.Data.Entity;
using Microsoft.AspNet.Identity;
using Microsoft.AspNet.Identity.EntityFramework;
using MoreDakka.Data;

namespace MoreDakka
{
    public class MyDbInitializer : DropCreateDatabaseAlways<BoardContext>
    {
        protected override void Seed(BoardContext context)
        {
            InitializeIdentityForEF(context);
            base.Seed(context);
        }

        private void InitializeIdentityForEF(BoardContext context)
        {
            var UserManager = new UserManager<ApplicationUser>(new UserStore<ApplicationUser>(context));
            var RoleManager = new RoleManager<IdentityRole>(new RoleStore<IdentityRole>(context));
            string name = "Admin";
            string password = "123456";

            //Create Role Admin if it does not exist
            if (!RoleManager.RoleExists(name))
            {
                RoleManager.Create(new IdentityRole(name));
            }

            var user = new ApplicationUser();
            user.UserName = name;
            var adminresult = UserManager.Create(user, password);

            //Add User Admin to Role Admin
            if (adminresult.Succeeded)
            {
                UserManager.AddToRole(user.Id, name);
            }
        }
    }
}